bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -767,11 +806,10 @@ bool ReplicaImp::validatePreProcessedResults(const PrePrepareMsg *msg) {
   errors.resize(error_id);
   for (auto err : errors) {
     if (err) {
-      // trigger view change
-      LOG_ERROR(VC_LOG,
-                "Received PrePrepareMsg containing PreProcessResult with invalid signature: "
-                    << *err << ". Ask to leave view " << getCurrentView());
-      askToLeaveView(ReplicaAsksToLeaveViewMsg::Reason::PrimarySentBadPreProcessResult);
+      // Indicate a view change
+      LOG_WARN(VC_LOG,
+               "Received PrePrepareMsg containing PreProcessResult with invalid signature"
+                   << KVLOG(*err) << " Replica will ask to leave view" << KVLOG(getCurrentView()));
       return false;
```

Comment:
:-) The idea was to have something like:
"Received PrePrepareMsg containing PreProcessResult with an invalid signature. The replica will ask to leave a view" << KVLOG(*err, getCurrentView()))

Reply:
Thanks, let me make that change and then checkin.